src/components/container/container.js

```diff
@@ -251,9 +251,13 @@ export default class Container extends UIObject {
     this.trigger(Events.CONTAINER_CONTEXTMENU, this, this.name)
   }
 
-  setCurrentTime(time) {
-    this.trigger(Events.CONTAINER_SEEK, time, this.name)
-    this.playback.seek(time)
+  setCurrentTime(time, inseconds) {
```

Comment:
I'm in favor of not having this flag, we can create another method to be called by `seekPercentage()`